src/partials/footer.html

```diff
@@ -1 +1,30 @@
-<footer>TEST</footer>
+<footer>
+  <div>
+    <div>HOMEMADE ICE CREAM MADE<span> WITH PASSION</span></div>
```

Comment:
Jest to niejako tytuł sekcji footer więc użyj tagu h2. W wytycznych chodzi o to by nie użyć h1, bo może być tylko jedno na stronie

Reply:
hm.. jest dokładnie taki zapis "W stopce znajduje się zduplikowany tytuł, nie trzeba go wyróżniać tagiem tytułu" - dla mnie h2 to właśnie tag tytułu, ale ok, zmienie.

Reply:
dobra, pewnie chodziło żeby nie używać <title> a nie nagłówka, pomyrdało mi się 🗡️

Reply:
być może, ale tekst w div to źle wygląda mimo wszystko